### DISCUSSION TITLE
Udemy Course is sufficient for docker certification ?

### DISCUSSION THREAD

**Discussion by itinfracode** [BretFisher/ama#154] [Q&A]
Hi Bret,

I am Pratik and I am currently pursuing your course "Docker Mastery".

This is really very well framed and greatly described docker course. I have completed this course almost 60%.

I am quite comfortable with all the terms and topics covered in this course.

I would like to certify with Docker and want to know is this course is sufficient to pass Docker certification exam ? Or I should learn more about Docker ?

If I still need to learn more, can you guide me where can educate and prepare myself for Docker certification ?

Thanks,
Pratik

**Comment by BretFisher** [comment]
My Docker Mastery course pre-dates the Docker Certified Associate (DCA) exam. It will teach you all the fundamentals, but the DCA is based on Docker Enterprise Edition, which I don't teach (yet).

[See my article for all the details and further resources for study](https://www.bretfisher.com/docker-certified-associate/).

**Comment by AustinTSchaffer** [comment]
Hi @BretFisher,

Would you recommend the DCA as a good career-advancing certification? Do you feel that there are effective ways to study for the exam even if you do not have access to an Enterprise Edition of Docker?

I love your UDemy course by the way. I feel that it got me about 50% of the way toward the certification, but the remaining topics feel like a huge uphill climb, especially since I'm currently the only person at my company that uses Docker as a part of my day-to-day.

Best,
Austin

Footnote: I read your blog post about the DCA and found the https://github.com/DevOps-Academy-Org/dca-prep-guide, like you suggested. I'm definitely going to check it out, but I'm wondering if any more resources have come out over the last year since you wrote the blog post (that you know of).

**Comment by BretFisher** [comment]
Hey Austin. 
* I think any cert in your field is worth doing, but if your company doesn’t value Docker, then they likely won’t value the cert. It just depends on your goals. If you’re looking for a role change or promotion inside your company, then you’ll need to do things and get certs that your boss values. If you’re looking to make a jump to a different org, then learn what they use if you can, and study up on those tools (look at their job postings and skills they are looking to hire for). In many cases I’d think a cloud certification and maybe OS certification (linux/windows) would be something you should have *before* you get a container orchestration certification.
* To me, DCA is about Enterprise Edition. You can download the eval edition and study that way, but trying to study without EE is like trying to pass a AWS cert without ever using their cloud.
* Glad you like the course! More on the way :)
* I haven’t looked for any new DCA study resources recently, sorry.
